src/sentry/web/frontend/debug/debug_incident_trigger_email.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import, print_function
+
+from sentry.models import User
+from django.views.generic import View
+
+from sentry.models import Organization, Project
+from sentry.snuba.models import SnubaQuery
+from sentry.incidents.action_handlers import generate_incident_trigger_email_context
+from sentry.incidents.models import (
+    Incident,
+    AlertRule,
+    AlertRuleTrigger,
+    TriggerStatus,
+    IncidentStatus,
+)
+
+
+from .mail import MailPreview
+
+
+class DebugIncidentTriggerEmailView(View):
```

Comment:
Isn't this the same as `DebugAlertRuleTriggerEmailView`? Although I suppose that needs to be updated to the new format.

Reply:
Ooof, was there really another that I completely missed